src/store/web3.ts

```diff
@@ -72,7 +72,7 @@ const getters = {
     return state.contractAddress.OTHER
   },
   isEthAccountConnected (state) {
-    return state.ethAddress === '' || state.ethAddress === 'undefined'
+    return !(state.ethAddress === '' || state.ethAddress === 'undefined')
```

Comment:
May it be undefined?

Reply:
Yes, but I'll check initial cause of undefined one more time.